Multisig/UI/Settings/App Settings/Owner Wallet Management/EnterSeedPhraseView.swift

```diff
@@ -31,25 +31,28 @@ struct EnterSeedPhraseView: View {
     var goNext = false
 
     var body: some View {
-        VStack(spacing: 24) {
-            Text("Enter your seed phrase")
-                .headline()
-            Text("Import the seed phrase (12 or 24 words) from your hardware or MetaMask owner wallet to sign transactions")
-                .body()
-                .multilineTextAlignment(.center)
-            EnterSeedView(seed: $seed, isEditing: $isEditing, isValid: $isValid, errorMessage: $errorMessage)
-            Spacer()
-            NavigationLink(destination: SelectOwnerAddressView(rootNode: rootNode, onSubmit: {
-                self.presentationMode.wrappedValue.dismiss()
-            }),
-                           isActive: $goNext,
-                           label: { EmptyView() }).isDetailLink(false)
-        }
-        .padding()
-        .navigationBarTitle("Import Wallet", displayMode: .inline)
-        .navigationBarItems(trailing: nextButton)
-        .onAppear {
-            self.trackEvent(.ownerEnterSeed)
+        NavigationView {
+            VStack(spacing: 24) {
+                Text("Enter your seed phrase")
+                    .headline()
+                Text("Import the seed phrase (12 or 24 words) from your hardware or MetaMask owner wallet to sign transactions")
+                    .body()
+                    .multilineTextAlignment(.center)
+                EnterSeedView(seed: $seed, isEditing: $isEditing, isValid: $isValid, errorMessage: $errorMessage)
+                Spacer()
+                NavigationLink(destination: SelectOwnerAddressView(
+                                rootNode: rootNode, onSubmit: {
+                                    self.presentationMode.wrappedValue.dismiss()
+                                }),
+                               isActive: $goNext,
+                               label: { EmptyView() })
+            }
+            .padding()
```

Comment:
will this make the scroll indicators inset 16 px on smaller devices? (we had this bug in another screen)